cmd/indexer/internal/actions/transaction_test.go

```diff
@@ -0,0 +1,150 @@
+package actions
+
+import (
+	"context"
+	"math/big"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/golang/mock/gomock"
+	"github.com/mailchain/mailchain/cmd/internal/datastore"
+	"github.com/mailchain/mailchain/cmd/internal/datastore/datastoretest"
+	"github.com/mailchain/mailchain/internal/address/addresstest"
+	"github.com/mailchain/mailchain/internal/protocols"
+	"github.com/mailchain/mailchain/internal/protocols/ethereum"
+	"github.com/pkg/errors"
+)
+
+var txHash = []byte("0x98beb27135aa0a25650557005ad962919d6a278c4b3dde7f4f6a3a1e65aa746c")
+var blockHash = []byte("0x373d339e45a701447367d7b9c7cef84aab79c2b2714271b908cda0ab3ad0849b")
```

Comment:
`[]byte("...")` will convert the string to bytes ignoring its encoding. The tests pass because checks `txHash` and `blockHash` are the same, it would be confusing for someone to read this file seeing these values without being decoded.

`encodingtest.MustDecodeHexZeroX` will decode the hex encoded string and panic is its invalid. This is acceptable for test file.

Reply:
Thanks @robdefeo , have fixed the tests now to use only test utils